README.md

```diff
@@ -1,12 +1,12 @@
 ![StreetComplete](http://www.westnordost.de/streetcomplete/featureGraphic.png)
 
-An Android app which finds wrong, incomplete or extendable data in the user's vicinity and provides the tools to complete these easily and directly on site without having to use another editor.
+StreetComplete is an easy to use editor of OpenStreetMap data available for Android. It can be used without any OpenStreetMap-specific knowledge. It asks simple questions, with answers directly used to edit and improve OpenStreetMap data.
```

Comment:
Maybe link to https://www.openstreetmap.org though? (In case someone discovers it, who does not know what OSM is?)